### PR TITLE
NETBEANS-3188 use Apache Maven 3.6.3

### PR DESCRIPTION
**Pull request by ebarboni**
Attemp to have last version of maven embedded

I don't understand why licence file contains so much htlm code.

I had duplicate issue. I'm not sure my configuration is ok

### CODE REVIEW

File: nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps [hunk 1]
```diff
@@ -59,11 +59,18 @@ enterprise/web.core.syntax/external/struts-tiles-1.3.10.jar enterprise/web.strut
...
+groovy/gradle/external/gradle-4.10.2-bin.zip java/maven.embedder/external/apache-maven-3.6.3-bin.zip
 groovy/gradle/external/gradle-4.10.2-bin.zip platform/api.annotations.common/external/jsr305-3.0.2.jar
 groovy/gradle/external/gradle-4.10.2-bin.zip platform/libs.junit4/external/hamcrest-core-1.3.jar
 groovy/gradle/external/gradle-4.10.2-bin.zip platform/libs.junit4/external/junit-4.12.jar
+groovy/gradle/external/gradle-4.10.2-bin.zip java/maven/external/aether-api-1.13.1.jar
+
+#
+#gradle-4.10.2/lib/javax.inject-1.jar in groovy/gradle/external/gradle-4.10.2-bin.zip and apache-maven-3.6.3/lib/javax.inject-1.jar in 
+#java/maven.embedder/external/apache-maven-3.6.3-bin.zip are identical
+#gradle-4.10.2/lib/plugins/plexus-cipher-1.7.jar in groovy/gradle/external/gradle-4.10.2-bin.zip and apache-maven-3.6.3/lib/plexus-cipher-1.7.jar in java/maven.embedder/external/apache-maven-3.6.3-bin.zip are identical</failure>
```
Comment (matthiasblaesing) [inline nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps]: This looks strange - if the ignore is needed, it should be present, if it is a comment, please remove it.

Comment (ebarboni) [reply nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps]: no this is bad copy paste sorry

**Comment by ebarboni** [comment]
> 
> 
> Looks sane to me - I would convert the license text from HTML to Plain text though. This affects:
> 
>     * apache-maven-3.6.3-bin-notice.txt
> 
>     * apache-maven-3.6.3-epl-license.txt
> 
>     * apache-maven-3.6.3-jsoup-license.txt
> 
>     * apache-maven-3.6.3-jsr-250-license.txt
> 
>     * apache-maven-3.6.3-slf4j-license.txt
> 
> 
> We are already required to review the notices and clean them up if needed, so I argue, that cleaning up the licenses is sane too.

I agree, but this particular license where htmlized.

I would also go for plain text and I they match non prefixed maven license maybe better to merge to one.

**Comment by ebarboni** [comment]
@matthiasblaesing some ' changes to " to get rid of Maven CDDL specific. I try to put plain text I hope it's ok.

**Comment by ebarboni** [comment]
shasqhed my side, force pushed, and will merge

**Pull request merged**